src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +29,12 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        from_log_id = self.request.GET.get('from_log')
+        message = LogEntry.objects.get(id=from_log_id).display()
```

Comment:
Also, we need to
* Check that the user can actually use this LogEntry, e.g. ``LogEntry.objects.get(id=from_log_id, event=self.request.event, action_type='pretix.plugins.sendmail.sent').display()`` to narrow the choice down so people cannot access texts from other events
* Check for errors, i.e.
    ````
    try:
        message = …
    except LogEntry.DoesNotExist:
        raise Http404(_('You suppliad an invalid log entry ID'))
    ````
    using a ``from django.http import Http404`` at the top of the file.